scripts/dev-template.mjs

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env zx
+
+import fs from 'fs/promises'
```

Comment:
can we add a comment to explain the reasoning for this script? It might not be straightforward if a new dev joins us why we have this and inside the template, there is also a dev command so it might be confusing which one to use.